.changelog/1488.trivial.md

```diff
@@ -1 +0,0 @@
-Move events cards to tabs
```

Comment:
#1488 (and extended with #1489) was user-facing, could manually re-add to full changelog